pkg/utils/stream_fields_parser.go

```diff
@@ -0,0 +1,395 @@
+package utils
```

Comment:
I don't think you need to have this parser. This parser is for logsql parsing, support logical operations, regexes, etc. 
I think stream fields just can be parsed as simple as comma-separated list of key-value pairs enclosed in curly braces: `{channel_id=\"CGZF1H6L9\",channel_name=\"general\"}`.

Reply:
updated parser simplified a bit of logic. Most of the cases are covered by VictoriaLogs. Just parse and check a bit of the correct content of the `_stream` field